src/main/java/seedu/intrack/model/internship/Website.java

```diff
@@ -0,0 +1,61 @@
+package seedu.intrack.model.internship;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.intrack.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Internship's address in the internship tracker.
```

Comment:
```suggestion
 * Represents an Internship's website in the internship tracker.
```
I think this line was missed when refactoring address to website.

Reply:
Thank you for your suggestion, the changes have been implemented.